prow/plugins.yaml

```diff
@@ -473,6 +470,9 @@ plugins:
   - require-matching-label
   - trigger
 
+  kubernetes/minikube:
+  - trigger
```

Comment:
/cc @donmccasland 
`kubernetes/minikube` has a presubmit ProwJob configured, but it was never triggered because the `trigger` plugin was not enabled. https://github.com/kubernetes/test-infra/blob/3f64bfde03dc54f21dee3c848f0149c66e420906/config/jobs/kubernetes/minikube/minikube.yaml#L16-L29

I don't think enabling this will cause any problems because the job is both optional and must be manually triggered.